examples/bri-3/src/bri/identity/bpiSubjects/api/subjects.controller.ts

```diff
@@ -27,13 +39,17 @@ export class SubjectController {
   }
 
   @Put()
```

Comment:
id should be part of the route and not dto to be restful i believe, same as you did for delete

Reply:
Fixed